kevm_pyk/src/kevm_pyk/solc_to_k.py

```diff
@@ -304,26 +299,29 @@ def gen_claims_for_contract(empty_config: KInner, contract_name: str, calldata_c
     return claims
 
 
-def contract_to_k(contract: Contract, empty_config: KInner, foundry: bool = False) -> Tuple[KFlatModule, Optional[KFlatModule]]:
-
-    contract_name = contract.name
+def contract_to_k(contract: Contract, empty_config: KInner, foundry: bool = False, exclude_tests: Iterable[str] = ()) -> Tuple[KFlatModule, Optional[KFlatModule]]:
 
     sentences = contract.sentences
-    module_name = contract_name.upper() + '-BIN-RUNTIME'
+    module_name = contract.name.upper() + '-BIN-RUNTIME'
     module = KFlatModule(module_name, sentences, [KImport('EDSL'), KImport('INT-SIMPLIFICATION'), KImport('LEMMAS')])
 
     claims_module: Optional[KFlatModule] = None
     contract_function_application_label = contract.klabel_method
     function_test_calldatas = []
-    for tm in contract.test_methods:
-        klabel = tm.production.klabel
-        assert klabel is not None
-        args = [abstract_term_safely(KVariable('_###SOLIDITY_ARG_VAR###_'), base_name=f'V{name}') for name in tm.arg_names]
-        calldata: KInner = KApply(contract_function_application_label, [KApply(contract.klabel), KApply(klabel, args)])
-        callvalue: KInner = intToken(0) if not tm.payable else abstract_term_safely(KVariable('_###CALLVALUE###_'), base_name='CALLVALUE')
-        function_test_calldatas.append((calldata, callvalue))
+    for tm in contract.methods:
+        if f'{contract.name}.{tm.name}' in exclude_tests:
+            _LOGGER.warning(f'Excluding test from contract {contract.name}: {tm.name}')
+        elif tm.name.startswith('testFail'):
+            _LOGGER.warning(f'Ignoring test from contract {contract.name}: {tm.name}')
```

Comment:
Is this preferable to not ignoring the `testFail` cases and just having them in the `exclude` file until the behavior for `testFail` is added? Maybe not, but just wanted to suggest this.

Reply:
I'm ignoring testFail for now because I think it needs a different generation process, so I just didn't want to commit to a particular way of encoding it until we actually know what it should look like.

Reply:
It's also just harder right now to hook up the `testFail` functionality. So I wanted to get the test harness in place first, and later iterations will worry about that.

Currently, all the `testFail` tests are listed in the `tests/foundry/exclude` file anyway, so we won't forget about them.